examples/pincell_depletion/restart_depletion.py

```diff
@@ -71,7 +73,7 @@
 
 days = 24*60*60
 plt.figure()
-plt.plot(time/days, keff, label="K-effective")
+plt.plot(time/days, keff[:, 0], label="K-effective")
```

Comment:
While we're here, maybe it would be better to make this an errorbar plot, as it sets a better example for new users of MC codes.

Reply:
Yeah, good call -- I'll change that